images/openshift-installer/Dockerfile.ci

```diff
@@ -1,28 +1,28 @@
-# This Dockerfile is a used as a CI job to validate Tectonic installer on CentOS
+# This Dockerfile is a used as a CI job to validate OpenShift installer on CentOS
 
 FROM openshift/origin-release:golang-1.9 as build
 WORKDIR /go/src/github.com/openshift/installer
 COPY . .
 ### Install Terraform
 ENV TERRAFORM_VERSION="0.11.1"
 # Install Terraform
-# TERRAFORM_URL enables us to build the upstream-terraform Tectonic builder
+# TERRAFORM_URL enables us to build the upstream-terraform OpenShift builder
```

Comment:
#35 is removing this comment entirely, altgough it will be asy to rebase #35 on top if you want to keep this change and this PR lands first.